src/tests/stdlib.rs

```diff
@@ -9,6 +12,20 @@ use tempfile::tempdir;
 use tempfile::TempDir;
 use std::process::{Command, Stdio};
 
+/*
+ * Autoload the Amber test files for stdlib and match the output with the output.txt file
+ */
+#[test_resources("src/tests/stdlib/*.ab")]
+fn amber_test(input: &str) {
+    let code = fs::read_to_string(input)
+        .expect(&format!("Failed to open {input} test file"));
```

Comment:
this should be omitted to an `include_str!` macro, it will load the file at compile time and throw an error if can't open

Reply:
the one drawback is that it will require to rewrite the whole thing to a `proc_macro` or something, or hardcode the entire thing